core/src/main/resources/static/management/user/buttonsAJAX.js

```diff
@@ -106,14 +106,58 @@ function otherCheck() {
     }
 }
 
+// Save button in editUserModal
+function updateUser(userId) {
+    clearAllErrorsSpan();
+    let formData = $('#editUserForm' + userId).serializeArray().reduce(function (obj, item) {
+        obj[item.name] = item.value;
+        return obj;
+    }, {});
+    validateForUpdate(formData, userId);
+    let returnData = {
+        "id": formData.id,
+        "name": formData.name,
+        "email": formData.email,
+        "role": formData.role,
+        "userCredo": formData.userCredo,
+        "userStatus": formData.userStatus
+    }
+    $.ajax({
+        url: '/management/users/',
+        type: 'PUT',
+        contentType: 'application/json',
+        dataType: 'json',
+        data: JSON.stringify(returnData),
+        success: function () {
+            history.go();
+        }
+    });
+}
+
+function validateForUpdate(formData, userId) {
+    let errorCounter = 0;
+    if(formData.name.length < 6 || formData.name.length > 30){
+        document.getElementById("errorModalUpdateName" + userId).innerText += "Name must contain between 6 and 30 characters";
+        errorCounter++;
+    }
```

Comment:
I would advice addding a more declarative way of performing the validation. Take a look here: https://getbootstrap.com/docs/5.0/forms/validation/